### PR TITLE
docs: improve and fix minor issues

### PR DESCRIPTION
**Pull request by qmonnet**
Provide minor improvements of the text or its formatting at various places of the documentation.

### CODE REVIEW

File: Documentation/gettingstarted/encryption-ipsec.rst [hunk 1]
```diff
@@ -17,24 +17,25 @@ as Cilium-managed host traffic, will be encrypted using IPsec. This guide uses
 Kubernetes secrets to distribute keys. Alternatively, keys may be manually
 distributed, but that is not shown here.
 
-Packets destined to the same node they were sent out of are not encrypted.
-This is an intended behavior as it would not provide any benefits because the
-raw traffic on the node can be seen.
+Packets are sent not encrypted when they are destined to the same node from
```
Comment (joestringer) [inline Documentation/gettingstarted/encryption-ipsec.rst]: Note, the changes in this file will conflict with #14940 CC @aditighag

**Review by joestringer — APPROVED** [review]

File: Documentation/gettingstarted/encryption-ipsec.rst (hunk 1 above)
Comment (aditighag) [reply Documentation/gettingstarted/encryption-ipsec.rst]: Ah, thanks @joestringer . @qmonnet #14940 already fixes some of these formatting related lines based on your review comments. :)

Comment (qmonnet) [reply Documentation/gettingstarted/encryption-ipsec.rst]: Oh that was February, I quite forgot about this PR. Fair enough, let's put mine on hold, I'll rebase once Aditi's merged. Thanks!

File: Documentation/operations/troubleshooting.rst [hunk 2]
```diff
@@ -218,7 +218,7 @@ Ensure Hubble is running correctly
 ----------------------------------
 
 To ensure the Hubble client can connect to the Hubble server running inside
-Cilium, you may use the ``hubble status`` command:
+Cilium, you may use the ``hubble status`` command, from a Cilium pod:
```
Comment (qmonnet) [inline Documentation/operations/troubleshooting.rst]: Will conflict with https://github.com/cilium/cilium/pull/15981, let's also wait that one is merged.

File: Documentation/gettingstarted/encryption-ipsec.rst (hunk 1 above)
Comment (rolinh) [reply Documentation/gettingstarted/encryption-ipsec.rst]: @qmonnet PR #15965 duplicated this paragraph to the wireguard guide for consistency. Could you apply the same change there on this other guide as well?

Comment (aditighag) [reply Documentation/gettingstarted/encryption-ipsec.rst]: Yes, indeed. :) the PR was on hold because there were some encryption fixes underway. Thanks!

**Review by rolinh — APPROVED** [review]

**Review by aditighag — APPROVED** [review]

**Pull request merged**